docs/data/toolpad/core/components/dashboard-layout/DashboardLayoutSidebarHidden.js

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+
+import { AppProvider } from '@toolpad/core/AppProvider';
+import Box from '@mui/material/Box';
+import { DashboardLayout } from '@toolpad/core/DashboardLayout';
+import PropTypes from 'prop-types';
+import Typography from '@mui/material/Typography';
+import { createTheme } from '@mui/material/styles';
+
+const NAVIGATION = [];
+
+const demoTheme = createTheme({
+  cssVariables: {
+    colorSchemeSelector: 'data-toolpad-color-scheme',
+  },
+  colorSchemes: { light: true, dark: true },
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 600,
+      md: 600,
+      lg: 1200,
+      xl: 1536,
+    },
+  },
+});
+
+function DemoPageContent({ pathname }) {
+  return (
+    <Box
+      sx={{
+        py: 4,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        textAlign: 'center',
+      }}
+    >
+      <Typography>Dashboard content for {pathname}</Typography>
+    </Box>
+  );
+}
+
+DemoPageContent.propTypes = {
+  pathname: PropTypes.string.isRequired,
+};
+
+function DashboardLayoutSidebarHidden(props) {
+  const { window } = props;
+
+  const [pathname, setPathname] = React.useState('/dashboard');
+
+  const router = React.useMemo(() => {
+    return {
+      pathname,
+      searchParams: new URLSearchParams(),
+      navigate: (path) => setPathname(String(path)),
+    };
+  }, [pathname]);
+
+  // Remove this const when copying and pasting into your project.
+  const demoWindow = window !== undefined ? window() : undefined;
+
+  return (
+    // preview-start
```

Comment:
You can remove `// preview-start` and `// preview-end` here and the code preview will show only the DashboardLayout which is the most important information.